.tekton/pipeline/upgrade-tests.yaml

```diff
@@ -90,7 +90,7 @@ spec:
         - name: repo_url
           value: $(params.repo_url)
         - name: revision
-          value: $(params.revision)
+          value: $(params.target_branch)
```

Comment:
I don't think that is the right fix. If changes are done to the script in the `ci` folder, they won't be tested unless the version checked out is the `revision`.

I believe the proper fix is to checkout the right branch before every test.

Reply:
the each git-clone task does checkout the right branch before every test, so the `revision` should align with the git-clone task, for example https://github.com/openshift-pipelines/pipeline-service/blob/main/.tekton/pipeline/upgrade-tests.yaml#L66